vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -155,13 +155,18 @@ ent prj install
 ent prj install --conflict-strategy=OVERRIDE
 ```
 
+  > **Note**
+  >
+  > In the event of a timeout or pod failure, refer to the errors captured by the cm and plugin deployer logs. 
+
 ## Use the App Builder
-1. Log into the `App Builder`
+1. Log into the App Builder
```

Comment:
No backticks on step 1 but yes on step 2?

Reply:
augh so we haven't been backticking App Builder anywhere... usually we say something like, "login to your App Builder and go to 'this` -> `that`"

maybe i'll combine steps here to mimic that and skirt the issue??

Reply:
@Lyd1aCla1r3 Works for me.  Sorry for the added noise.